canonical/ubuntu.com
# Copy update: /confidential-computing

## Done

- Copy update

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Make sure that the new and updated sections aren't reflected to the HTML file
- Resolve the "new section" comments once approved

## Issue / Card

Fixes WD-14726

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
